mozilla/addons-server
# Expose all the reasons why there is a due_date to the extensions queue (queryset)

fixes: mozilla/addons#14946 to enable mozilla/addons#14921



### Description

Splits all the reasons that a version may get a due_date into `Q` objects, and adds a `VersionManager` method to return them.  The same Q objects are then used to calculate `should_have_due_date` and to annotate `Addon.get_queryset_for_pending_queues`, so they can be exposed as flags in the review queue.

Some of the (seemingly) unused reviewer tools icons were repurposed.

### Context

To show flags for why an add-on is in the review queue we need to show all the reasons, so to be consistent we should use the exact same queries. 

### Testing

- set up some add-ons with combinations of NeedsHumanReview reasons, and reasons they may need a pre-review.
- see all the relevant flags shown in the reviewer tools.
- bonus tests for weird combinations of nhrs on different versions that are inactive, or the versions disabled, etc.

### Checklist
